docs/command/atlas-kubernetes-dry-run.txt

```diff
@@ -0,0 +1,84 @@
+.. _atlas-kubernetes-dry-run:
```

Comment:
Do we have any tool or linting that check these docs for syntax issues? or can we see a rendering of the doc somehow?

Reply:
oh, I guess these are auto generated right?

Reply:
Yes, this is autogenerated by `make gen-docs` from the command description